api/openapi-spec.yaml

```diff
@@ -0,0 +1,473 @@
+          type: integer
+          format: int32
+          default: 1
+      - name: shallow
+        in: query
+        description: "Determines if external ER's should be used to compile\
+          \ the results of query. Use `false` to use External ER's."
+        schema:
+          type: boolean
+          default: false
+      - name: lazy
+        in: query
+        description: "If lazy is `true`, it implies that when the events limit\
+          \ is reached according to pazesize no additional request is performed and the search will stop."
+        schema:
+          type: boolean
+          default: false
+      - name: readable
+        in: query
+        description: |
+          Determines if event time should be in milliseconds, or in Human readable time format. Ex:
+
+          `false` 1499076742982
+
+          `true`  2018-10-31T13:36:00.824Z.
+        schema:
+          type: boolean
+          default: false
+      - name: params
+        in: query
+        description: |
+          To search for specific events or artifacts, filtering with parameters is supported.
+
+          **Syntax:**
+
+          `<resource>?key[.key ...]<FC>value[&key[.key ...]<FC>value ...]`
+
+          <FC> is one of the Filter comparators described below. To traverse into nested structures and filter on their keys, namespacing with
+          `.` (dot) is used.
+
+          ```
+          =  - equal to
+          >  - greater than
+          <  - less than
+          >= - greater than or equal to
+          <= - less than or equal to
+          != - not equal to
+          ```
+
+          **Examples, single key:**
+
+          `/events/?meta.type=EiffelActivityStartedEvent`
+
+          **Examples multiple keys:**
+
+          ```
+          /events/?key1=value1&key2=value2
+          /events/?key1>=value1&key2!=value2&key3<value3    #Multiple comparators
+          /events/?key1>value1&key1<=value2&key1!=value3    #Multiple constraints on one key
+          ```
+
+          **Examples nested structures:**
+          ```
+          /events/?data.identity=pkg:maven/my.namespace/my-name@1.0.0
+          /events/?meta.source.domainId=my.domain&data.identity=pkg:maven/my.namespace/my-name@1.0.0    #Multiple keys and nested structures
+          ```
+
+          Note that mutiple keys only is allowed with logical AND (via `&`). There are no support for logical OR.
+
+          Filters also allow for regex to be used. This means you can make more complex matchings or use partial matching. In regex `.*` matches anything and basically works like wildcards.
+
+          **Examples of regex:**
+          ```
+          /events/?data.identity=my-artifact@1.0.0              #Partial match using artifact name and version
+          /events/?data.identity=my.namespace                   #Partial match using artifact namespace
+          /events/?data.identity=my.namespace/.*@1.0.0          #Matches any version 1.0.0 artifact within the namespace 'my.namespace'
```

Comment:
Change the query to /events?data.identity=my.namespace/.*.@1.0.0

Reply:
Fixed in a9debeb